telegram/service.go

```diff
@@ -22,11 +22,14 @@ type Service interface {
 	SendDocumentToGroup(ctx context.Context, document []byte, extension string, group int64) error
 
 	SendKeyboard(ctx context.Context, buttons []string, text string, chat int64) (int, error)
+	SendKeyboardGroup(ctx context.Context,buttons []string, text string, chat uint32) (error)
 	RegisterCommand(command Command)
 	RegisterCommandLet(commandlet Commandlet)
 
 	requestAuthorisation(chat uint32, name string) (string, error)
 	pollAuthorisation(token string) (uint32, error)
+	SetState(user int, chat uint32, state string, field []string) error
```

Comment:
Cannot expose the internal state - the commandlet already has the Fields and Next State methods, so there should be no reason to have this. Instead, we should probably update the Command Interface to have a Next State field as well and store that.